.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
@aditianshu We don't need these changes. Get rid of it.

Reply:
Removed